viewing_party/party.py

```diff
@@ -1,23 +1,152 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    if title == None or genre == None or rating == None:
+        new_movie = None
+    else:
+        new_movie = {"title": title, "genre": genre, "rating": rating}
+    return new_movie
 
+def add_to_watched(user_data, movie):
+    watched_list = user_data["watched"]
+    watched_list.append(movie)
+    user_data["watched"] = watched_list  

```

Comment:
If we delete/comment this line out, we can actually see that the tests still pass-- we don't need to re-assign `user_data["watched"] = watched_list`. That's because `watched_list` is a _reference_ to `user_data["watched"]`-- anything we change on `watched_list` will change on `user_data["watched"]`, because they're both pointing to the same thing. Let me know if you have questions on this!